README.md

```diff
@@ -13,26 +13,47 @@ There is not much documentation on how to get started. I will create issues that
 
 ## Running & debugging with Visual Studio 2022
 
-- Install Docker
-- Install Visual Studio 2022
-- Set docker-compose as the Startup Project
-- Start Debugging
+1. Install Docker
+2. Install Visual Studio 2022
+3. Set docker-compose as the Startup Project
+4. Start Debugging
 
 ## Running without Visual Studio 2022
 
-There is no documentation as of yet, but you should use `docker-compose` command. Feel free to fill-in this info in a PR.
+There is no documentation yet, but you should use `docker-compose` command. Feel free to fill-in this info in a PR.
 
 ## How to make UT4 connect to new master server?
 
-- Use `Files/Engine.ini` in this repository as a template
-- Replace all occurances of `<PUT_MASTER_SERVER_DOMAIN_HERE>` with the domain of master server you want to connect to
-- Copy the content and append all of it to your local `Engine.ini` file
-  - Windows path: `C:\Users\your_username\Documents\UnrealTournament\Saved\Config\WindowsNoEditor`
+1. Use `Files/Engine.ini` in this repository as a template
+2. Replace all occurrences of `<PUT_MASTER_SERVER_DOMAIN_HERE>` with the domain of master server you want to connect to
+3. Copy the content and append all of it to your local `Engine.ini` file
+    - Windows path: `C:\Users\your_username\Documents\UnrealTournament\Saved\Config\WindowsNoEditor`
+4. Register new account in `POST account/api/create/account` endpoint
+5. Now you have 2 options:
+   1. Option 1 **(recommended)**:
+      1. Add `UnrealTournament` parameter to your UT4 shortcut:
+      Example:
+
+      ```text
+      "D:\Epic Games\UnrealTournament\Engine\Binaries\Win64\UE4-Win64-Shipping.exe" UnrealTournament
+      ```
+
+      2. After starting game, you should see login window. Enter your credentials and you should be able to play.
+   2. Option 2:
+      1. Get `access_token` from `POST account/api/oauth/token` with `grant_type: password`
+      2. `GET account/api/oauth/exchange` with Bearer authorization (`access_token` from `POST account/api/oauth/token`)
+      3. Add parameters to your UT4 shortcut and replace `your_password_code` with `code` from `GET account/api/oauth/exchange`:
+
+      ```text
+      "D:\Epic Games\UnrealTournament\Engine\Binaries\Win64\UE4-Win64-Shipping.exe" UnrealTournament -AUTH_LOGIN=unused -AUTH_PASSWORD=your_password_code -AUTH_TYPE=exchangecode
+      ```
+
+      NOTE: You need to repeat point 2 and 3 if you want to login again.
```

Comment:
Due to indentation, list bullets are numbers on 1st level, roman numerals on 2nd level and letters on 3rd level. 
```suggestion
      NOTE: You need to repeat point *b* and *c* if you want to login again.
```

Reply:
Thanks